cli/tests/pcluster/validators/test_cluster_validators.py

```diff
@@ -2725,3 +2726,43 @@ def test_get_capacity_reservation(self, queue, compute_resource, expected_respon
         assert_that(DictLaunchTemplateBuilder().get_capacity_reservation(queue, compute_resource)).is_equal_to(
             expected_response
         )
+
+
+@pytest.mark.parametrize(
+    "encryption_settings, expected_error_message",
+    [
+        (
+            [
+                ("queue1", True),
+                ("queue2", False),
+            ],
+            "The Encryption parameter of the root volume of the queue queue2 is not consistent with the "
+            "value set for the queue queue1. Please be sure this is expected.",
+        ),
+        (
+            [
+                ("queue1", True),
+                ("queue2", True),
+            ],
+            None,
+        ),
+        (
+            [
+                ("queue1", False),
```

Comment:
nit: you can save some lines by removing the last comma:
```python
          ([("queue1", False), ("queue2", True)], None),
          ([("queue1", True), ("queue2", False)], None),
```

then we can add the `False, True case too`.